keyboards/alhenkb/macropad5x4/readme.md

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
This file should follow the style of the [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md). Eg. bootloader instructions and `make` commands.